app/src/main/java/org/p2p/wallet/common/feature_toggles/remote_config/AppFirebaseRemoteConfig.kt

```diff
@@ -20,28 +24,30 @@ class AppFirebaseRemoteConfig : RemoteConfigValuesProvider {
         get() = FirebaseRemoteConfig.getInstance()
 
     fun loadRemoteConfig(onConfigLoaded: (featureToggles: Map<String, String>) -> Unit) {
-        // do not put it in release - we don't need to change fetch interval there
-        if (BuildConfig.DEBUG) {
-            remoteConfig.setConfigSettingsAsync(createRemoteConfigSettings())
-                .addOnSuccessListener { Timber.d("Remote config for debug is set") }
-                .addOnFailureListener { Timber.e("Remote config for debug is not set", it) }
-        }
-
-        // maybe add loading screen in the start for remoteConfig to finish the job?
-        remoteConfig.fetchAndActivate()
-            .addOnSuccessListener {
-                isFetchFailed = false
-                Timber.d("Remote config fetched and activated")
-                Timber.i("Remote config fetched toggles: ${allFeatureTogglesRaw()}")
-
-                onConfigLoaded.invoke(allFeatureTogglesRaw())
+        appScope.launch {
```

Comment:
it's better to launch this task in the place where it called

1) AppCreatedAction already has AppScope
2) Fetching remote config is async task by default, launch is redundant